html/elements/link.json

```diff
@@ -719,6 +720,13 @@
             "__compat": {
               "description": "Alternative stylesheets.",
               "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Alternative_style_sheets",
+              "spec_url": [
+                "https://html.spec.whatwg.org/multipage/links.html#rel-alternate",
+                "https://html.spec.whatwg.org/multipage/links.html#the-link-is-an-alternative-stylesheet",
```

Comment:
Are all of these really needed? Is the first one sufficient maybe?

Reply:
> Are all of these really needed? Is the first one sufficient maybe?

I think only the first one is needed — so I’ve updated the patch to make that so